products/rabbitmq.md

```diff
@@ -91,4 +91,4 @@ releases:
 > [RabbitMQ](https://www.rabbitmq.com/) is an open source message broker written in Erlang that supports multiple messaging protocols.
 
 - **General Support**: includes regular patch releases
-- **Extended Support**: includes security patches, and high-severity issues reported by users with a commercial license.
+- **[Extended Support](https://tanzu.vmware.com/rabbitmq)**: includes security patches, and fixes for high-severity issues reported by users with a commercial license from VMWare.
```

Comment:
maybe we need to add link to `from VMWare` this part instead of `Extended Support` part

Reply:
@captn3m0 ^^

Reply:
Extended Support is a VMWare offering, so linking the former makes sense. Will remove "From VMWare" from the latter, it's just confusing.